yt/docs/en/_includes/admin-guide/install-ytsaurus.md

```diff
@@ -289,7 +289,8 @@ Events:
 Follow all required steps to [start the {{product-name}} cluster](#starting-cluster). Then install the chart:
 
 ```
-helm upgrade --install ytsaurus-ui github/ytsaurus-ui/packages/ui-helm-chart/
+git clone https://github.com/ytsaurus/ytsaurus-ui.git
```

Comment:
it was not obvious that chart installation  was coming from source code
i used https url for clone which may be not convenient for some users, but still it's more clear that to do to install chart